www/js/services.js

```diff
@@ -48,12 +48,13 @@ angular.module('pvta.services', ['ngResource'])
 
 
 .factory('StopList', function(){
-  var stopsList = [];
-  var pushToList = function(stop){
-    stopsList.push(stop);
-  };
+  var stopsList = []; 
+
   var pushEntireList = function(list){
-    stopsList = stopsList.concat(list);
+    list = _.uniq(list, true, 'Name');
+    _.each(list, function(stop){
```

Comment:
Since `list` and `stopsList` are both arrays, any reason why we need to loop though it again?  We should just be able to say ` stopsList = list`